src/opnsense/mvc/app/controllers/OPNsense/Diagnostics/Api/SystemhealthController.php

```diff
@@ -131,17 +131,11 @@ public function getSystemHealthAction($rrd = "", $inverse = 0, $detail = -1)
                 $response['set']['count'] = count($records);
                 $response['set']['step_size'] = $response['sets'][$detail]['step_size'];
                 foreach ($records as $key => $record) {
-                    $record['area'] = true;
-                    if ($inverse && $key % 2 != 0) {
-                        foreach ($record['values'] as &$value) {
-                            $value[1] = $value[1] * -1;
-                        }
-                    }
                     $response['set']['data'][] = $record;
                 }
             }
             for ($i = 0; $i < count($response['sets']); $i++) {
-                unset($response['sets'][$detail]['ds']);
+                unset($response['sets'][$i]['ds']);
```

Comment:
I'm not sure this was even the intention to be honest, the only reason these sets exists is to determine the granularity, it might be better to remove the "sets" for now. The question is if currently different granularities even exists, if not, a fixed set (as currently implemented) my be fine.